<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mySettingDialog</class>
 <widget class="QDialog" name="mySettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始实验</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>801</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    font: 25px &quot;黑体&quot;;
	background-color: rgb(240, 240, 240);
    border-radius:6px;
    margin-top:12px;
    border-width: 1px;
    border-style: solid;
    border-color: rgb(255, 0, 0);
}
QGroupBox:title {
    color:rgb(28, 120, 167);
    subcontrol-origin: margin;
    left: 10px;
}</string>
   </property>
   <property name="title">
    <string>数据库</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据库地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDirget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
